airbyte-integrations/connectors/source-sendgrid/unit_tests/unit_test.py

```diff
@@ -48,13 +44,11 @@ def test_source_wrong_credentials():
 
 
 def test_messages_stream_request_params(mock_pendulum_now):
-    stream = get_stream("messages")
+    start_time = 1558359837
+    stream = Messages(start_time)
     state = {"last_event_time": 1558359000}
-    expected_params = {
-        "query": 'last_event_time BETWEEN TIMESTAMP "2019-05-20T06:30:00Z" AND TIMESTAMP "2021-12-31T16:00:00Z"',
-        "limit": 1000,
-    }
-    request_params = stream.retriever.request_params(
-        stream_state=state, stream_slice={"start_time": "2019-05-20T06:30:00Z", "end_time": "2021-12-31T16:00:00Z"}
+    request_params = stream.request_params(state)
+    assert (
```

Comment:
This is awful. The assertion fails when we run a github action because the timezone on one's laptop is different from the one when the test is running on the EC2 instance. Still looking into the best way to work around this